@xen-orchestra/web/src/components/pool/PoolHeader.vue

```diff
@@ -50,9 +51,8 @@ const { pool } = defineProps<{
   pool: XoPool
 }>()
 
-const { t } = useI18n()
-
 const router = useRouter()
+const { t } = useI18n()
 
 const goToNewVm = () => router.push({ name: '/vm/new', query: { poolid: pool.id } })
```

Comment:
I know it's not your code but if you can change to RouterLink it will be good 

```suggestion
  <RouterLink :to="{ name: '/vm/new', query: { poolid: pool.id } }">
        <UiButton :left-icon="faPlus" variant="secondary" accent="brand" size="medium">
          {{ t('new-vm') }}
        </UiButton>
      </RouterLink>
```